README.md

```diff
@@ -197,10 +201,12 @@ Since Aug 2020 Chrome has a new [secure-by-default model](https://blog.chromium.
 There are two ways to use `oidc-server-mock` with this change.
 
 1. Run the container with HTTPS enabled (see above).
-2. Change cookies `SameSite` mode from default `None` to `Lax`. To do so just add the following to `ASPNET_SERVICES_OPTIONS_INLINE/PATH` env var:
+2. Change cookies `SameSite` mode from default `None` to `Lax`. To do so just add the following to `SERVER_OPTIONS_INLINE` (or the file at `SERVER_OPTIONS_PATH`):
 
-```json
+```javascript
```

Comment:
why?

Reply:
That's just to have comments, otherwise they appear in bright red.